.devops/pagopa-code-review-pipelines.yml

```
@@ -73,7 +73,7 @@ resources:
     - repository: checkout-be-mock
       type: github
       name: pagopa/pagopa-checkout-be-mock
-      ref: master
+      ref: CHK-4442-payment-methods-handler-api
```

```suggestion
      ref: master
```
TODO revert after https://github.com/pagopa/pagopa-checkout-be-mock/pull/111 merge